live-examples/css-examples/basic-user-interface/appearance.html

```diff
@@ -0,0 +1,39 @@
+<section id="example-choice-list" class="example-choice-list">
```

Comment:
This is missing `data-property` but I'm not sure what we should put for it. Maybe we should omit it for now, and when the editor can handle prefixing properties we could use `appearance`.

Reply:
That was my thinking with omitting it in the first place. I'll leave it for now.